opcua/ua/uatypes.py

```diff
@@ -30,12 +30,26 @@
 FILETIME_EPOCH_AS_DATETIME = datetime(1601, 1, 1)
 
 
+class UTC(tzinfo):
+    """
+    UTC
+    """
+
+    def utcoffset(self, dt):
+        return timedelta(0)
+
+    def tzname(self, dt):
+        return "UTC"
+
+    def dst(self, dt):
+        return timedelta(0)
+
+
+
 # method copied from David Buxton <david@gasmark6.com> sample code
 def datetime_to_win_epoch(dt):
     if (dt.tzinfo is None) or (dt.tzinfo.utcoffset(dt) is None):
-        dt = dt.replace(tzinfo=pytz.utc)
-    else:
-        dt = dt.astimezone(tz=pytz.utc)
+        dt = dt.replace(tzinfo=UTC())
```

Comment:
hmm, I'm running a test with orora-gawler connection but where the datetime being pulled here is a string.